src/scenes/AcceleratorRouter/NavBar.tsx

```diff
@@ -65,7 +68,7 @@ export default function NavBar({ backgroundTransparent, setShowNavBar }: NavBarP
         id='overview'
         label={strings.OVERVIEW}
         onClick={() => closeAndNavigateTo(APP_PATHS.ACCELERATOR_OVERVIEW)}
-        selected={!!isAcceleratorOverviewRoute}
+        selected={!!isOverviewRoute || !!isScoringRoute || !!isVotingRoute || !!isVotingEditRoute}
```

Comment:
@nickgraz we need this to also include scoring route matches

Reply:
Oh right, I'll add it for now. Nice catch.